samples/MqttSample/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mcr.microsoft.com/dotnet/aspnet:5.0-buster-slim AS build-env
```

Comment:
Is this file not redundant given that we have another in the root of the solution?

Reply:
Removed